ansible/srm-toolchain/prometheus/alerts.rules

```diff
@@ -3,7 +3,7 @@ groups:
   rules:
 
   - alert: high_bandwidth
-    expr: storage_bandwidth > 80
+    expr: storage_requests > 80
```

Comment:
both bandwidth and throughput are replaced by requests

Reply:
Thanks for pointing this out. Pushed the fixes in the next commit.